DetectorDescription/DDCMS/plugins/dd4hep/DDAngular.cc

```diff
@@ -35,7 +35,7 @@ static long algorithm(Detector& /* description */, cms::DDParsingContext& ctxt,
   Volume mother = ns.volume(args.parentName());
   string childName = args.value<string>("ChildName");
   if (strchr(childName.c_str(), NAMESPACE_SEP) == nullptr)
-    childName = ns.name() + childName;
+    childName = ns.prepend(childName);
```

Comment:
The line above should be removed since ```ns.prepend``` does this comparison to avoid adding a namespace to a name that already has it. Yet another reason to forbid a blind ```+``` operation :-)